skfda/exploratory/visualization/_boxplot.py

```diff
@@ -249,30 +245,55 @@ class Boxplot(FDataBoxplot):
 
     """
 
-    def __init__(self, fdatagrid, depth_method=ModifiedBandDepth(), prob=[0.5],
-                 factor=1.5):
+    def __init__(
+        self,
+        fdatagrid: FDataGrid,
+        depth_method: Optional[Depth[FDataGrid]] = None,
+        prob: Tuple[float] = (0.5,),
```

Comment:
The type hint should be `Sequence[float]`. Note also that in order to accept tuples of arbitrary length the type should have been `Tuple[float, ...]`.

Reply:
The type should be `Sequence[float]`.